examples/wait_group/main.neva

```diff
@@ -0,0 +1,12 @@
+import { sync }
+
+component Main(start) (stop) {
+  nodes { wg sync.WaitGroup, Println }
+  :start -> [
+    (3 -> wg),
+    ('Hello' -> println -> (-1 -> wg)),
+    ('Neva' -> println -> (-1 -> wg)),
+    ('World!' -> println -> (-1 -> wg))
+  ]
+  wg -> :stop
+}
```

Comment:
Read the description. It highly likely -1 reaches the wg before 3, which normally would cause the program to stop, but I had to add a protection to the component to guard against this.

Reply:
> Read the description

Sorry, I'll be more careful

> It highly likely -1 reaches the wg before 3, which normally would cause the program to stop, but I had to add a protection to the component to guard against this.

Ok but I feel like this problem could be solved by having 2 ports